### ISSUE TITLE
Should be able to self-test this workflow locally

### ISSUE THREAD

**Issue by Bucknalla** [product-os/docusaurus-builder#52]
Currently there's no mechanism to see how this container works locally; we should have an example directory to be used for self testing to ensure that plugins, such as mermaid and tsdoc (in the future), are still working.

**Comment by ab77** [comment]
Yes, ideally the workflow should create a Docusaurus CF pages site from its README. md

**Comment by vipulgupta2048** [comment]
@ab77 We do that already: https://docusaurus-builder.pages.dev/
